docs-devsite/firestore_.md

```diff
@@ -2551,6 +2552,18 @@ Union type from all supported SDK cache layer.
 export declare type FirestoreLocalCache = MemoryLocalCache | PersistentLocalCache;
 ```
 
+## ListenSource
+
+Describe the source a query listens to.
+
+`default`<!-- -->: listens to both cache and server changes `cache`<!-- -->: listens to changes in cache only
```

Comment:
Set to `default` to listen to both cache and server changes. Set to `cache` to listen to changes in cache only.